src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -23,24 +23,35 @@ internal class Instrumenter
         private readonly string[] _includeFilters;
         private readonly string[] _excludedFiles;
         private readonly string[] _excludedAttributes;
+        private readonly bool _singleHit;
         private InstrumenterResult _result;
         private FieldDefinition _customTrackerHitsFilePath;
         private FieldDefinition _customTrackerHitsArray;
         private FieldDefinition _customTrackerHitsMemoryMapName;
+        private FieldDefinition _customTrackerSingleHit;
         private ILProcessor _customTrackerClassConstructorIl;
         private TypeDefinition _customTrackerTypeDef;
         private MethodReference _customTrackerRegisterUnloadEventsMethod;
         private MethodReference _customTrackerRecordHitMethod;
         private List<string> _asyncMachineStateMethod;
 
-        public Instrumenter(string module, string identifier, string[] excludeFilters, string[] includeFilters, string[] excludedFiles, string[] excludedAttributes)
+        public Instrumenter(string module, string identifier, string[] excludeFilters, string[] includeFilters, string[] excludedFiles, string[] excludedAttributes, bool singleHit)
         {
             _module = module;
             _identifier = identifier;
             _excludeFilters = excludeFilters;
             _includeFilters = includeFilters;
             _excludedFiles = excludedFiles ?? Array.Empty<string>();
             _excludedAttributes = excludedAttributes;
+            _singleHit = singleHit;
+        }
+
+        private bool IsCoreLibrary
+        {
+            get
+            {
+                return Path.GetFileNameWithoutExtension(_module) == "System.Private.CoreLib";
```

Comment:
this will be cached by https://github.com/tonerdo/coverlet/pull/308/files right?I mean is this tmp code?

Reply:
This pull request builds on #308. This change is handled in the other pull request.